docs/docs/02-installation/06-debuntu.md

```
@@ -13,10 +13,10 @@ The script will download and install all dependencies (except for Ollama), insta
 
 The script functions as an update script in addition to an installer. See **[Updating](#updating)**.
 
-### 1. Download the script from the [Karakeep repository](https://github.com/absolutepraya/karakeep/blob/main/karakeep-linux.sh)
+### 1. Download the script from the [Karakeep repository](https://github.com/absolutepraya/marka/blob/main/karakeep-linux.sh)
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)